### PR TITLE
ReadTheDocs Initial Documentation Setup

### LINKED ISSUE #140: Update README with badges
### Description
We want to add badges to our README similar to what other repos have. Some of them are dependent on having the project available on PyPi, so we are waiting to go public before adding them.

### Resources
https://shields.io/
[Example README](https://github.com/pydantic/pydantic)

### LINKED ISSUE #144: Move non-Scryfall (maybe non-Magic?) enums out of catalogs
Maybe put them back into a scooze.enums or something?

### LINKED ISSUE #153: ReadTheDocs or similar documentation site

### LINKED ISSUE #253: Update Security.md
Add a link to the security tab of this repo in the security.md file.

### LINKED ISSUE #255: Update People Documentation
Update docs/people.md similar to how Pydantic or other repositories handle a maintainers file.

### LINKED ISSUE #256: Update CLI Documentation
Add docs for the CLI. There will probably be a lot of writing to be done here beyond just relying on mkdocstrings because so much of it is produral.

### LINKED ISSUE #257: Update db_interface Documentation
The docs/db_interface markdown files are pretty bare bones because we didn't have a clear direction for them when they were written. This is a ticket meant to revisit those files, where they live, and their contents.

### LINKED ISSUE #258: Add License Badge to README
We have other badges, but for some reason the LICENSE badge wasn't playing nice. Once the docs are in dev, we can go ahead and give this another try.

```
[![license](https://img.shields.io/github/license/arcavios/scooze.svg)](https://github.com/arcavios/scooze/blob/dev/LICENSE)
```

### PR DESCRIPTION
**Pull request by iambroadband**
Starting this out as a draft PR so we can discuss. You can preview these changes here: https://scooze.readthedocs.io/en/latest/

You can also checkout this branch and run `mkdocs serve` to make changes on your local.

### CODE REVIEW

File: pyproject.toml
Comment (iambroadband) [inline pyproject.toml:1]: Sorting dependencies alphabetically here.

**Comment by iambroadband** [comment]
Did a self review. Here are the files I think still need some form of attention:

```
docs/changelog.md
docs/dataclasses/deck.md
docs/db_interface/python/bulkdata.md
docs/db_interface/python/scooze.md
docs/db_interface/rest/routers.md
docs/extras/cli.md
docs/index.md
docs/models/deck.md
docs/people.md
mkdocs.yml
```

File: SECURITY.md
Comment (iambroadband) [inline SECURITY.md:1]: Removing this from the top level. I am fairly certain that once we have setup the SECURITY.md for the security tab the first time, it's stored somewhere on GitHub. If we ever need to make changes to it, we can do it through the UI.

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+nav:
+    - Get Started:
+          - Introduction to scooze: index.md
+          - Installation: installation.md
+          - Setup: setup.md
+          - Contributing: contributing.md
+          - Changelog: changelog.md
+    - API Documentation:
+          - Models:
```
Comment (fjork3) [inline mkdocs.yml:60]: Do we want data classes first here? Those seem more likely what people will be working with directly over the DB models.

**Review by fjork3 — APPROVED** [review]

**Pull request merged**